content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/tutorials/plc-ide-setup-license/content.md

```diff
@@ -100,21 +100,54 @@ Click OK to save the settings and press the connect button on **On-line > connec
 
 ![Connect to the board](assets/connect.png)
 
-### 6. License Activation
+### 6. License Activation With Product Key (Portenta Machine Control)
 
 If the communication is successful the main panel will show the license section to be filled in and a status indicator.
 
-To use your product license, fill it in on the form and press the **Activate** button.
+#### Online Activation
+
+To use your product license, paste the key in the blank next to **Product Key:** and press the **Activate** button.
 
 ![Connection success](assets/licenseActivation.png)
 
-After you click the **Activate** button, it will use the license and attach it to the board. A popup telling you to reboot the target (which is the device connected to the PLC IDE) will show up once the attachment finishes. Press the reset button of your device to perform the reboot.
+After you click the **Activate** button, the IDE will bind the license to the hardware ID. A popup telling you to reboot the target (which is the device connected to the PLC IDE) will show up: press the reset button of your device to reboot.
 
 ![Activation succeed](assets/connectedAndActivated.png)
 
-Once the board has been activated, it is recommended to get the **Hardware-ID** of your device. 
+Once the product has been activated, it is recommended to get the **Hardware-ID** of your device. 
+
+#### Offline Activation
```

Comment:
the #### rendering is not creating a breakpoint between the tittle and the text. I'm going to investigate.

Reply:
This is a common thing across so many tutorials, we can skip this problem for now